docs/wiki/deployment/configuration.md

```diff
@@ -429,6 +429,8 @@ The columns, and their values, will be appended to each log line as follows. Ass
 
 Expect the normal set of log keys to be included and note that `decorations` is a top-level key in the log line whose value is an embedded map.
 
+The configuration flag `top_level_decorations`  can be set to `true` to make decorator data populate as top level key/value objects instead of being contained as a child of `decorations`.  When using this feature, you must be weary of key collisions in your data.
```

Comment:
Can you expand on "collisions in your data"? Maybe "collisions in existing, reserved, top-level keys"?

It would be awesome to provide an example of what this does, perhaps a _with_ and _without_ example log line.